iOSClient/Utility/NCUtility.swift

```diff
@@ -496,19 +496,21 @@ class NCUtility: NSObject {
         return  UIImage(named: "file")!.image(color: color, size: size)
     }
     
-    func loadUserImage(for user: String, displayName: String?, urlBase: String) -> UIImage {
-        var image: UIImage?
-        
-        let fileName = String(CCUtility.getUserUrlBase(user, urlBase: urlBase)) + "-" + user + "-original.png"
+    func loadUserImage(for user: String, displayName: String?, userUrlBase: String, original: Bool = false) -> UIImage {
+
+        //@marino: I'm not even sure the original is needed.. seems to work just fine with the 'normal' one
```

Comment:
@marinofaggiana can you confirm this?

Reply:
I don't remember exactly this func (maked it several time ago) but seems works, yes.